docs/volume-operations/volume-operations.md

```diff
@@ -0,0 +1,167 @@
+
+### Expanding PV
+To expand the capacity of a PV, you should modify its corresponding PVC. Once done, the newly generated PVs will be created with the new capacity. 
+
+**Prerequisites**
+- Volume expansion support for PVC is in Kubernetes 1.11 or above, which is already required in [Prerequisites](#prerequisites).
+- The StorageClass must set `allowVolumeExpansion` to true. The default StorageClass `iomesh-csi-driver` already does this. If a StorageClass is created and configured with custom parameters, verify that its `allowVolumeExpansion` is set to "true". 
+
+
+**Procedure**
+
+1. Create and configure the YAML file. The following example assumes a PVC named `example-pvc` with a capacity of `10Gi`.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: example-pvc
+spec:
+  storageClassName: iomesh-csi-driver-default
+  accessModes:
+    - ReadWriteOnce
+  resources:
+    requests:
+      storage: 10Gi # The original capacity of the PVC.
+```
+
+2. Run the following command to apply the YAML file. 
+
+```bash
+kubectl get pvc example-pvc
+```
+
+After running the command, you should see an example like:
+
+```output
+NAME          STATUS   VOLUME                                     CAPACITY    ACCESS MODES   STORAGECLASS                AGE
+example-pvc   Bound    pvc-b2fc8425-9dbc-4204-8240-41cb4a7fa8ca   10Gi        RWO            iomesh-csi-driver-default   11m
+```
+
+3. Set the field `storage` to a new value.
+
+```yaml
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+name: example-pvc
+spec:
+  storageClassName: iomesh-csi-driver-default
+  accessModes:
+    - ReadWriteOnce
+resources:
+  requests:
+    storage: 20Gi # Enter a new capacity value.
+```
+
+4. Run the following command to apply the new YAML file.
+
+```bash
+kubectl apply -f example-pvc.yaml
+```
+
+5. Run the following command to check the capacity of the PVC.
+
+```bash
+kubectl get pvc example-pvc 
+```
+
+After running the command, you should see an example below:
+
+```output
+NAME          STATUS   VOLUME                                     CAPACITY    ACCESS MODES   STORAGECLASS                AGE
+example-pvc   Bound    pvc-b2fc8425-9dbc-4204-8240-41cb4a7fa8ca   20Gi        RWO            iomesh-csi-driver-default   11m
+```
+
+6. Run the following command to check the PV capacity.
+   
+```bash
+kubectl get pv pvc-b2fc8425-9dbc-4204-8240-41cb4a7fa8ca 
+```
+
+After running the command, you should see an example like this:
+```output
+NAME                                       CAPACITY   RECLAIM POLICY   STATUS   CLAIM                 STORAGECLASS
+pvc-b2fc8425-9dbc-4204-8240-41cb4a7fa8ca   20Gi       Retain           Bound    default/example-pvc   iomesh-csi-driver-default
+```
+
+### Cloning PV
+A clone is a duplicate of an existing volume in the system and data on the source will be duplicated to the destination. To clone a PV, you should specify an existing PVC in the field `dataSource` so that you can clone a volume based on it.
```

Comment:
```suggestion
A clone is a duplicate of an existing volume in the system and data on the source will be duplicated to the destination. To clone a PV, you should create a new PVC and specify an existing PVC in the field `dataSource` so that you can clone a volume based on it.
```